relay/kusama/src/lib.rs

```diff
@@ -1677,7 +1677,7 @@ construct_runtime! {
 		ParaSessionInfo: parachains_session_info = 61,
 		ParasDisputes: parachains_disputes = 62,
 		ParasSlashing: parachains_slashing = 63,
-		OnDemandAssignmentProvider: parachains_assigner_on_demand = 64,
+		OnDemand: parachains_assigner_on_demand = 64,
```

Comment:
This renaming requires a storage migration using https://github.com/paritytech/polkadot-sdk/blob/604f56f03db847a90aa4fdb13be6b80482a4dcd6/substrate/frame/support/src/storage/migration.rs#L323
As storage keys will change

Reply:
@tdimitrov Can you look into that please?